docs/powergate/ffs.md

```diff
@@ -0,0 +1,108 @@
+# FFS
+
+The FFS the manages all the necessary state and capabilities to provide multi-tiered file storage through the Powergate. The FFS is the primary API for storing and retrieving data, tracking long-term deals on Filecoin, and allowing data persisted on Filecoin to be available on IPFS. To start using the FFS APIs you must first create an _FFS Instance_.
+
+## FFS Instance
+
+The FFS API is scoped to one or a set of Filecoin wallet addresses. So to start accessing the FFS API, you must init a new instance at which time the Powergate will:
+
+1. Create a new default wallet address for the FFS Instance.
```

Comment:
It also funds the new address with some FIL

Reply:
+ (optionally)

Reply:
done